OxfordBodleian/Ullendorff/BDLeu2.xml

```diff
@@ -224,10 +226,17 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 
                     <history>
                         <origin>
-                            <origDate notBefore="1954" notAfter="1955" evidence="internal-date"/>
+                            <origDate notBefore="1954" notAfter="1955"
+                                evidence="internal-date">1954/55</origDate>
+                            The manuscript was produced in 1954/55 by the scribe <persName ref="PRS14000WaldaA" role="scribe"/>
```

Comment:
Have we decided not to include the EC date in cases like this?

Reply:
I don't think it was discussed, I've added it for now